chia/wallet/util/compressed_types.py

```diff
@@ -0,0 +1,71 @@
+from dataclasses import dataclass
+
+from chia.types.blockchain_format.program import Program
+from chia.types.coin_spend import CoinSpend
+from chia.types.spend_bundle import SpendBundle
+from chia.util.streamable import Streamable, streamable
+from chia.wallet.util.puzzle_compression import PuzzleCompressor
+
+
+@dataclass(frozen=True)
+@streamable
```

Comment:
I doubt we need these to be streamable, so I'd recommend removing this unless necessary.

Reply:
I assume we want these stored on TransactionRecords and the like.  We can serialize to bytes first, but why bother when we can type it this way?